Applications/LLaMA/jni/main.cpp

```diff
@@ -1,17 +1,20 @@
 // SPDX-License-Identifier: Apache-2.0
 /**
- * Copyright (C) 2023 Seungbaek Hong <sb92.hong@samsung.com>
+ * Copyright (C) 2023 Jihoon Lee <sb92.hong@samsung.com>
```

Comment:
The copyright is Seunbaek Hong. Not Jihoon Lee.

Reply:
Fix, Thank you for Review